styles.css

```diff
@@ -1101,3 +1101,20 @@ textarea {
   background: #2230d2;
   color: #fff;
 }
```

Comment:
- [ ] [OPTIONAL] You did great adding validation but I think it would be better to use `red` color for your validation message to be clear and this color can easily show users that the email is not valid 😉